src/transformers/convert_slow_tokenizer.py

```diff
@@ -897,7 +897,7 @@ class T5Converter(SpmConverter):
     def vocab(self, proto):
         num_extra_ids = self.original_tokenizer._extra_ids
         vocab = [(piece.piece, piece.score) for piece in proto.pieces]
-        vocab += [(f"<extra_id_{i}>", 0.0) for i in range(num_extra_ids - 1, -1, -1)]
+        vocab += [(f"<extra_id_{i}>_", 0.0) for i in range(num_extra_ids - 1, -1, -1)]
```

Comment:
How to fix this bug in LlamaTokenizerFast? I changed to new tokenizer and now fast and slow work different. Do you have any hack method like this to make fast and slow work as the same, thanks!

Reply:
@ArthurZucker